src/views/nav.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import {Link} from "react-router-dom";
+
+export default function NavView() {
+	return (
+		<div>
+			<Link to="/">Home</Link>
+			<Link to="/login">Login</Link>
+		</div>
+	);
+}
```

Comment:
We need the login component and '/' view only, all these extra components are not needed. Please review the scope of this task and undo all the unrelated work you have done. Ensure to always stick to the scope of the task, it is very important.

Reply:
@LehruAngela I have made the change